Packages/com.unity.inputsystem/InputSystem/State/InputState.cs

```diff
@@ -30,7 +30,7 @@ public static class InputState
         /// </summary>
         public static uint updateCount => InputUpdate.s_UpdateStepCount;
 
-        public static double currentTime => InputRuntime.s_Instance.currentTime + InputRuntime.s_CurrentTimeOffsetToRealtimeSinceStartup;
+        public static double currentTime => InputRuntime.s_Instance.currentTime - InputRuntime.s_CurrentTimeOffsetToRealtimeSinceStartup;
```

Comment:
Ouch... that's an embarrassing one... first reaction was "no way" :)

But... needs test.